src/Common/src/Interop/Windows/shell32/Interop.ShellExecuteExW.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Shell32
+    {
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        internal unsafe struct SHELLEXECUTEINFO
+        {
+            public uint cbSize;
+            public uint fMask;
+            public IntPtr hwnd;
+            public char* lpVerb;
+            public char* lpFile;
+            public char* lpParameters;
+            public char* lpDirectory;
+            public int nShow;
+            public IntPtr hInstApp;
+            public IntPtr lpIDList;
+            public IntPtr lpClass;
+            public IntPtr hkeyClass;
+            public uint dwHotKey;
+            public IntPtr hIcon;
```

Comment:
Nit: it looks like this field is actually a union, either hIcon and hMonitor, both IntPtrs.  We might want to name this field to reflect that, e.g. hIconOrhMonitor, or hIconMonitor, or something like that.